packages/loader/container-loader/src/deltaQueue.ts

```diff
@@ -115,6 +132,11 @@ export class DeltaQueue<T> extends TypedEventEmitter<IDeltaQueueEvents<T>> imple
         if (!this.paused) {
             this.ensureProcessing();
         }
+        if (this.logger) {
+            this.logger.sendTelemetryEvent({
+                eventName: "DeltaQueueSystemResume",
+            },  `${this.sysPause}`);
```

Comment:
nit: it does not need to be string, number will work just fine.
Having small comment on why we use this weird format might be useful for future reader (i.e. trying to force stack collection)